moveit_demo_nodes/run_move_group/launch/run_move_group.launch.py

```diff
@@ -98,14 +102,28 @@ def generate_launch_description():
                                  output='both',
                                  parameters=[robot_description])
 
-    # Fake joint driver
-    fake_joint_driver_node = Node(package='fake_joint_driver',
-                                  executable='fake_joint_driver_node',
-                                  parameters=[{'controller_name': 'panda_arm_controller'},
-                                              os.path.join(get_package_share_directory("run_move_group"), "config", "panda_controllers.yaml"),
-                                              os.path.join(get_package_share_directory("run_move_group"), "config", "start_positions.yaml"),
-                                              robot_description]
-                                  )
+    # Fake ros2_controller system
+    ros2_control_node = Node(
+        package='controller_manager',
+        executable='ros2_control_node',
+        parameters=[robot_description,  os.path.join(get_package_share_directory("moveit_resources_panda_moveit_config"), "config", "panda_ros_controllers.yaml")],
+        output={
+            'stdout': 'screen',
+            'stderr': 'screen',
+        },
+    )
+
+
+    # load joint_state_controller
+    load_joint_state_controller = ExecuteProcess(cmd=['ros2 control load_start_controller joint_state_controller'], shell=True, output='screen')
+    load_controllers = [load_joint_state_controller]
+    # load panda_arm_controller
+    load_controllers += [ExecuteProcess(cmd=['ros2 control load_configure_controller panda_arm_controller'],
```

Comment:
Should be removed once https://github.com/ros-controls/ros2_control/pull/310 is merged